### PR TITLE
Update helm ac script with revert feature

### PR DESCRIPTION
**Pull request by xpuska513**
## What
* Possible fix for ```mv: cannot stat 'charts/airbyte/Chart.yaml': No such file or directory```

### CODE REVIEW

**Review by cpdeethree — APPROVED** [review]

File: tools/bin/acceptance_test_kube_helm.sh [hunk 1]
```diff
@@ -77,7 +97,11 @@ if [ -n "$CI" ]; then
   # trap "mkdir -p /tmp/kubernetes_logs && write_all_logs" EXIT
 fi
 
-kubectl expose $(kubectl get po -l app.kubernetes.io/name=server -o name) --name exposed-server-svc --type NodePort --overrides '{ "apiVersion": "v1","spec":{"ports": [{"port":8001,"protocol":"TCP","targetPort":8001,"nodePort":8001}]}}'
+if [ -n "$CI" ]; then
+  kubectl expose $(kubectl get po -l app.kubernetes.io/name=server -o name) --name exposed-server-svc --type NodePort --overrides '{ "apiVersion": "v1","spec":{"ports": [{"port":8001,"protocol":"TCP","targetPort":8001,"nodePort":8001}]}}'
+else
+  kubectl port-forward svc/ab-airbyte-server 8001:8001 &
```
Comment (jdpgrailsdev) [inline tools/bin/acceptance_test_kube_helm.sh:103]: It appears that the service is named `svc/airbyte-airbyte-server-svc`, not `svc/ab-airbyte-server`.

**Review by cpdeethree — APPROVED** [review]

**Pull request closed without merging**